pkg/collector/blockio_energy.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+)
+
+var (
+	kblockdPodName       = "kworker-kblockd"
```

Comment:
looks like this a process instead of independent pod?

Reply:
it is a name i want to give to all the kblockd worker threads, but you are right, the name shouldn't be exported and confuse end users.

Reply:
We could have a prefix for OS related process `os-process-kworker-kblockd`